openlibrary/macros/EditionNavBar.html

```diff
@@ -3,7 +3,7 @@
 <ul class="work-menu sticky">
   <li class="selected">
     <a href="#editions-list">
-      $ungettext('View 1 Edition', 'View %(count)d Editions', edition_count, count=edition_count)
+      $ungettext('View %(count)s Edition', 'View %(count)s Editions', edition_count, count=edition_count)
```

Comment:
This is backwards. The correct fix is to make sure that `edition_count` is an integer so that the pluralization works.

Reply:
@tfmorris Pluralizations worked before this change too, and `edition_count` comes from `work.edition_count` which is an integer

Reply:
Your statement is inconsistent with the reported behavior. `%d` is the correct format code for integers (and `work.edition_count` is not mentioned anywhere in the macro). Since it's a macro, you should consider the possibility that some code is calling it correctly and other code calling it incorrectly.